### PR TITLE
Small improvements 

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
Styling fix + getting rid of thread.sleep calls where they're unnecessary.

### CODE REVIEW

File: source/geod24/LocalRest.d [hunk 1]
```diff
@@ -1576,7 +1579,9 @@ unittest
     assert(to_node.sleepFor(40) == 42);
 
     assertThrown!Exception(to_node.sleepFor(2000));
-    Thread.sleep(2.seconds);  // need to wait for sleep() call to finish before calling .shutdown()
+    to_node.ctrl.withTimeout(Duration.zero,  // wait for the node to wake up
```
Comment (Geod24) [inline source/geod24/LocalRest.d]: Can we keep the timeout to 2 seconds though ?

Comment (AndrejMitrovic) [reply source/geod24/LocalRest.d]: Wouldn't that suffer from timing issues? Maybe 3 would be better?

Comment (Geod24) [reply source/geod24/LocalRest.d]: Well, so would the previous version of the code, no ? But 3 works for me too, or even higher. I just prefer to have a timeout, as having an infinite wait when a test doesn't pass it not very developer-friendly.

Comment (AndrejMitrovic) [reply source/geod24/LocalRest.d]: Indeed. It's a good catch. I've updated it.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**